modules/apps/27-interchain-accounts/ibc_module_test.go

```diff
@@ -195,7 +208,6 @@ func (suite *InterchainAccountsTestSuite) TestOnChanOpenAck() {
 }
 
 func (suite *InterchainAccountsTestSuite) TestOnChanOpenConfirm() {
-	suite.SetupTest() // reset
```

Comment:
suite.SetupTest gets called before every `Test...` function. We need it when doing a for loop on test cases